src/MoBi.Core/Domain/Model/MoBiSimulation.cs

```diff
@@ -122,6 +133,24 @@ public IReadOnlyList<IBuildingBlock> BuildingBlocks()
          return buildingBlocks;
       }
 
+      public IReadOnlyList<ParameterValue> OriginalQuantityValues => _quantityValueCache.ToList();
```

Comment:
I'd suggest to returnd IReadOnlyDictionary or another type in this case which is IEnumerable too so that you don't have to ToList all the time.

Reply:
IReadOnlyCollection